src/__tests__/initializeCacheClient.test.js

```diff
@@ -0,0 +1,45 @@
+import initializeCacheClient from '../initializeCacheClient';
+
+const mockBind = jest.fn();
+const mockDisabled = jest.fn();
+const mockEnabled = jest.fn();
+
+const mockConfig = {
+    config: {
+        cache: {
+            enabled: true,
+            memcached: {
+                hosts: 'localhost:11211,localhost:11212',
+                maxExpiration: 1000,
+            },
+        },
+    },
+    terminal: {
+        disabled: mockDisabled,
+        enabled: mockEnabled,
+    },
+};
+
+jest.mock('@globality/nodule-config', () => ({
+    getContainer: () => (mockConfig),
+    getMetadata: () => ({ testing: false }),
+    bind: (key, fn) => mockBind(key, fn),
+}));
+
+describe('initializeCacheClient', () => {
+    it('binds a cache client to the graph', () => {
+        initializeCacheClient();
+        expect(mockBind).toHaveBeenCalledWith('cache', expect.anything());
```

Comment:
Can we have a test for the way that we can access the cache client from the graph?
We like to use tests as a usage example.